kubernetes/start

```diff
@@ -71,5 +76,23 @@ fi
 sudo mkdir -p $PWD/tmp/mariadb-socket
 sudo chmod 777 $PWD/tmp/mariadb-socket
 
+if docker images | grep "^sles12/velum" &> /dev/null ; then
+  log "Velum development image is present"
```

Comment:
can't we make a rebuild happen here in this script once the production image in the registry has changed?